packages/lwc-language-server/src/decorators/index.ts

```diff
@@ -46,6 +46,7 @@ export interface WireDecoratorTarget {
     params: { [name: string]: string };
     static: any;
     type: DecoratorTargetType;
+    adapter?: unknown;
```

Comment:
This field is not used within `lwc-language-server`. However, it _is_ present in the old metadata.

We want to ensure that mapped metadata and old metadata are identical. To that end, we've added the field here to allow its inclusion in the mapped metadata.